docs/docs.trychroma.com/markdoc/content/cloud/collection-forking.md

```diff
@@ -0,0 +1,80 @@
+# Collection Forking
+
+**Collection forking enables instant, zero-copy collection branching in Chroma Cloud.**
+
+Forking lets you create a new collection from an existing one instantly, using copy-on-write under the hood. The forked collection initially shares its data with the source and only incurs additional storage for incremental changes you make afterward.
+
+{% Banner type="tip" %}
+**Forking is available in Chroma Cloud only.** The file system on single-node Chroma does not support forking — see [Single-Node Chroma: Performance and Limitations](../guides/deploy/performance). Chroma Cloud uses block storage that enables true copy-on-write semantics.
```

Comment:
1. Linking out to performance and limitations seems a bit odd.
2. "Chroma Cloud uses block storage that enables true copy-on-write semantics." -> This doesn't quite make sense to me